apps/docs/docusaurus.config.js

```diff
@@ -69,23 +69,29 @@ module.exports = {
           //   position: 'left',
           // },
           {
-            docId: 'cdk/api/transformation-helpers/index',
+            docId: 'cdk/cdk',
             label: 'CDK',
             position: 'left',
             type: 'doc',
           },
           {
-            docId: 'state/getting-started/overview',
+            docId: 'state/state',
             label: 'State',
             position: 'left',
             type: 'doc',
           },
-          // {
-          //   docId: 'template/index',
-          //   label: 'Template',
-          //   position: 'left',
-          //   type: 'doc',
-          // },
+          {
+            docId: 'template/template',
+            label: 'Template',
+            position: 'left',
+            type: 'doc',
+          },
+          {
+            docId: 'eslint-plugin/eslint-plugin',
+            label: 'ESLint-Plugin',
```

Comment:
suggestion: Keep top navbar labels short and simple.
```suggestion
            label: 'ESLint',
```